src/util/pointer_predicates.h

```diff
@@ -50,6 +50,8 @@ exprt object_upper_bound(
   const exprt &pointer,
   const exprt &access_size);
 
+exprt validate_all_derefs(const exprt &ptr, const namespacet &);
```

Comment:
Add a comment block.

Reply:
We should move this to `contracts/utils.h`.

Reply:
`s/validate_all_derefs/validate_all_dereferences/g`

Reply:
Done.